gapis/api/templates/api.go.tmpl

```diff
@@ -249,10 +249,11 @@ import (
   type {{$name}} struct {
     Map *map[{{$key}}]{{$value}}
     a   arena.Arena
+    uncompareable
```

Comment:
Are we sure we never want to compare map references?

Reply:
It's a good question - I believe it to be completely valid to use the equality operator on a map in the API language.
What exactly is this fixing? Do you have a situation where two maps are in two different arenas or something?

Reply:
> Make maps uncomparable, they can't be map keys

So this is the reason, right? How come they can't be map keys?